WDL/runtime/config_templates/default.cfg

```diff
@@ -97,6 +97,9 @@ ignore_query = false
 enable_patterns = ["*"]
 disable_patterns = ["*.php", "*.aspx"]
 
+[outputs_cache]
+XDG_CACHE_HOME="~/.cache"
```

Comment:
If we're making a bunch of .json files, then we should probably put them in a miniwdl-specific subdirectory there so that the user can easily delete them all at once if desired. (SQLite would create just one file)

setting name could be just [call_cache] dir = ... or [call_cache] db = ...

Reply:
👍